src/assets/sass/protocol/components/_call-out.scss

```diff
@@ -8,11 +8,12 @@
 // Call Out component
 
 .mzp-c-call-out {
-    background-color: get-theme('background-color-alt');
+    background-color: $color-marketing-gray-20;
```

Comment:
This one did not end up with a theme variable.

Reply:
I think it would also be okay to standardize on the lighter background colour.

Reply:
I did start on adding an accent color to themes but don't want to pile more into this PR. We can think more about accent colors another time.